Adding clarification about EMU users being able to view public repositories on github.com (#12070)
github/docs: content/admin/authentication/managing-your-enterprise-users-with-your-identity-provider/about-enterprise-managed-users.md

{% data variables.product.prodname_managed_users_caps %} can only contribute to private and internal repositories within their enterprise and private repositories owned by their user account. {% data variables.product.prodname_managed_users_caps %} have read-only access to the wider {% data variables.product.prodname_dotcom %} community.

* {% data variables.product.prodname_managed_users_caps %} cannot create issues or pull requests in, comment or add reactions to, nor star, watch, or fork repositories outside of the enterprise.

> * {% data variables.product.prodname_managed_users_caps %} can view the code in public repositories on broader github.com, but cannot push code to repositories outside of the enterprise.

Reviewer: ```suggestion
* {% data variables.product.prodname_managed_users_caps %} can view code in public repositories hosted on github.com, but cannot push code to repositories outside of the enterprise.
```

Reviewer: `view` or `access`?

Reviewer 2: I don't think there's a difference from GitHub perspective in relation to code. However, now I think whether we need to be more clear that the users will be able to view/access not just code, but also things like PRs, issues, etc. Something along the lines `can access public repositories hosted on github.com`

Reviewer: agreed, @andrekolodochka - clarifying that for all intents and purposes that they have read access to not just the code in a public repo is a good idea. It will help counter a lot of the questions we get in this area.

Reviewer 3: ```suggestion
* {% data variables.product.prodname_managed_users_caps %} can view all public repositories on {% data variables.product.prodname_dotcom_the_website %}, but cannot push code to repositories outside of the enterprise.
```
